word2gauss/embeddings.pyx

```diff
@@ -166,21 +180,21 @@ cdef class GaussianEmbedding:
     cdef gradient_t gradient_func
 
     # pointers to data arrays
-    cdef DTYPE_t* mu_ptr
-    cdef DTYPE_t* sigma_ptr
-    cdef DTYPE_t* acc_grad_mu_ptr
-    cdef DTYPE_t* acc_grad_sigma_ptr
+    cdef DTYPE_t*mu_ptr
```

Comment:
This formatting change is both unnecessary for Python 3 support and kind of ugly.